lib/typescript/httpclient/model/RenderedContent.ts

```diff
@@ -0,0 +1,13 @@
+import {MessageState, SenderType} from './Message';
+
+export interface RenderedContent {
+  id: string;
+  content: any;
+  deliveryState?: MessageState;
+  senderType?: SenderType;
+  sentAt?: Date;
+}
+
+export function isFromContact(message: RenderedContent) {
+  return message?.senderType === SenderType.sourceContact;
+}
```

Comment:
To render templates inside the templates modal, we needed to tweak the render library so that it renders both messages and templates that are not messages. 

I've created a type for the Render library called _RenderedContent_ which works for both messages and templates. I think this solution is not the most  ideal (as we lose some type checking) and the naming could also be improved, but I couldn't think of anything else

Reply:
Probably already Content is a better name?

Reply:
Good idea. This is a hint that the render library is too message-specific. An ideal solution would be to extract the rendering logic that is specific to messages (sender information) to a wrapper and have it only expose content rendering logic. 

But that can be done in a separate issue I think.

Reply:
The type is also strange though as it seems still message specific. I will read the whole thing more carefully then

Reply:
agree on the first part @chrismatix but honestly wouldn't wait for a separate issue unless we pick it up immediately (which means I'd do here if it was for me)

Reply:
yeah I thought about Content, but feel that Content could be confusing, as it doesn't reflect that the type is exclusive to the render library (but yeah RenderedContent is not great)